Sources/TripKit/categories/NSNumber+Formatter.swift

```diff
@@ -18,22 +18,39 @@ extension NSNumber {
     formatter.roundingMode = .up
     return formatter
   }()
+    
+  /// formats the NSNumber to a readable currency formatted string that auto handles the decimal places.
+  public func toMoneyString(currencyCode: String) -> String {
+    let formatter = moneyFormatter(with: currencyCode)
+    formatter.locale = .current
+    
+    return formatter.string(from: self)!
+  }
   
+  /// formats the NSNumber to a readable currency formatted string that forces the number of decimal places.
   public func toMoneyString(currencyCode: String, decimalPlaces: Int = 0) -> String {
```

Comment:
I am surprised that this compiles. Which method will be used when calling `toMoneyString(currencyCode: "AUD")`. Doesn't that match both methods?

Reply:
I've disambiguated it as it no longer compiled when I added a second parameter with a default value to the first method.